### PR TITLE
feat(lambda): .net core 3.1 runtime

### PR DESCRIPTION
**Pull request by michaeldimoudis**
### Commit Message

Closes https://github.com/aws/aws-cdk/issues/7104

### End Commit Message

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by michaeldimoudis** [comment]
> 
> 
> Please rebase on master (to get these changes: #7024) and add in-code documentation instead of adding the linting exception :)

No problem, this has been done now :)

**Comment by michaeldimoudis** [comment]
Hi @nija-at I saw you approved a similar PR, are you able to have a look at this PR please? Thank you.

File: packages/@aws-cdk/aws-lambda/lib/runtime.ts [hunk 1]
```diff
@@ -112,6 +112,11 @@ export class Runtime {
    * The .NET Core 2.1 runtime (dotnetcore2.1)
    */
   public static readonly DOTNET_CORE_2_1 =  new Runtime('dotnetcore2.1',  RuntimeFamily.DOTNET_CORE);
+
```
Comment (nija-at) [inline packages/@aws-cdk/aws-lambda/lib/runtime.ts:115]: Remove trailing whitespace.

Comment (michaeldimoudis) [reply packages/@aws-cdk/aws-lambda/lib/runtime.ts]: That's done.

**Comment by wmkDev** [comment]
Hey guys, sorry if this is not the right place or if its a simple (or dumb) question, but how/when the package on "nuget / https://www.nuget.org/packages/Amazon.CDK.AWS.Lambda/" gets updated with the new runtime?

**Comment by nija-at** [comment]
We have a weekly release cadence for the CDK. The new runtime will be available on our next release, later this week.

**Pull request merged**